tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -29,12 +29,35 @@
 raw_intensity = mne.io.read_raw_nirx(fnirs_raw_dir, verbose=True).load_data()
 
 
+###############################################################################
+# View location of sensors over brain surface
+# -------------------------------------------
+#
+# Here we validate that the location of sources-detector pairs and channels
+# are in the expected locations. Source-detector pairs are shown as lines
+# between the optodes, channels (the mid point of source-detector pairs) are
+# shown as dots.
+
+mne.datasets.fetch_fsaverage()
```

Comment:
usually you do `fetch_fsaverage` or `sample.data_path()`, you should not need both. For this example I'd go with `sample` since it's more generally useful for learning MNE

If for some reason you do need `fetch_fsaverage`, it should go after the `data_path()` call and then the `subjects_dir` should be passed to `fetch_fsaverage`. But I don't think you should need it...

Reply:
Correct. It built fine when removed below